### DISCUSSION TITLE
SQL editor in the UI

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2277] [Feature Ideas]
We could add a SQL editor to the Mathesar UI, allowing users to run SQL queries directly from the web interface.

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by jeromeperez** [comment]
For example to :

- add foreign key between tables of different relation
- use functions like 'now' in default value of field or regular expression
- concatenate multiple columns in one
